### PR TITLE
fix rss feed to export full content of blog

### LINKED ISSUE #79: RSS feed of blog is broken
**What happened**:

RSS feed of blog is broken : https://falco.org/blog/index.xml

**What you expected to happen**:

Gather a XMLfile with full articles of blog

**How to reproduce it (as minimally and precisely as possible)**:

Just access to https://falco.org/blog/index.xml

**Anything else we need to know?**:

N/A

### PR DESCRIPTION
**Pull request by nevermosby**
Signed-off-by: nevermosby <legendarilylwq@gmail.com>

**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area blog

**What this PR does / why we need it**:

This PR is aim to fix issue #79 ,  thus, the rss feed can export full content of blog.

**Which issue(s) this PR fixes**:

Fixes #79  

**Special notes for your reviewer**:

N/A

### CODE REVIEW

**Comment by nevermosby** [comment]
Hi @kris-nova could u pls help review this pr?

**Comment by lucperkins** [comment]
RSS feeds are not supposed to display the full content of blog posts in the description field. Their core function is to provide notifications that a blog has been updated with new content. I would advise not accepting this change.

File: themes/falco-fresh/layouts/blog/section.rss.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
       {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
       <description>{{ .Summary | html }}</description>
+      <content>{{ .Content | html }}</content>
```
Comment (lucperkins) [inline themes/falco-fresh/layouts/blog/section.rss.xml]: There is no `content` field in the RSS spec: https://validator.w3.org/feed/docs/rss2.html.

Again, the purpose of RSS feeds is not to transmit the entire contents of a blog. It is, rather, to notify RSS-compliant reader clients of updates.

**Comment by lucperkins** [comment]
Closing due to non-compliance with the [RSS specification](https://validator.w3.org/feed/docs/rss2.html).

File: themes/falco-fresh/layouts/blog/section.rss.xml (hunk 1 above)
Comment (nevermosby) [reply themes/falco-fresh/layouts/blog/section.rss.xml]: `content` belongs to `atom` xml ns, which is declared on the top of this file, shall we use it?

**Comment by nevermosby** [comment]
> Closing due to non-compliance with the [RSS specification](https://validator.w3.org/feed/docs/rss2.html).

As this pr is closed due to some rss rules, is it necessary to  keep #79 open?

**Comment by lucperkins** [comment]
@nevermosby You're right: the Atom spec does allow for a `<content>` field. I've re-opened this.

**Comment by fntlnz** [comment]
Thanks for this @nevermosby !

**Pull request merged**